aws/lambda/log-classifier/src/bedrock/prompts.rs

```diff
@@ -0,0 +1,36 @@
+// LOG_SNIPPET a part of an error log
+pub const FIND_ERROR_LINE_PROMPT: &str = r#"
+You are an AI assistant tasked with identifying the most likely actionable error in a log snippet. Your goal is to find and report the single line that best indicates the error occurring in the system.
+
+Here is the log snippet you need to analyze:
+
+<log_snippet>
+{{LOG_SNIPPET}}
+</log_snippet>
+
+To complete this task, follow these steps:
+
+1. Carefully read through the entire log snippet.
+2. Look for lines that indicate errors, exceptions, failures, or unexpected behavior. Pay special attention to:
+   - Error messages
+   - Exception traces
+   - Warning messages
+   - Unexpected state changes
+   - Timeouts or performance issues
+3. If multiple error indicators are present, prioritize the one that seems most critical or is likely the root cause of other issues.
```

Comment:
Do you think we could update the prompt here to capture the logic where the relevant errors usually appear near the end of the log snippet?

Reply:
oops I don't think so personally. In that case we'd just shrink the snippet no?

Reply:
Ah ok, shrinking the snippet makes sense and it seems easier to implement.  IMO, there is also different ways to shrink the logs:

* A fix number of last N lines
* Output from the last n commands, i.e. we have that in the failure context https://github.com/pytorch/test-infra/pull/4649

I wonder if this is flexible enough to handle tough cases like `keep-going` or how inductor benchmark is done where a model could fail further up the logs and the job ignores it and continues with other models.